src/api/fileApi.ts

```diff
@@ -44,10 +45,30 @@ export const createMultipartUpload = async (file: UppyFile) => {
   return createMultipartUploadResponse.data;
 };
 
-export const createResourceAndMultipartUpload = async (file: UppyFile, onCreateFile: (resourceId: string) => void) => {
+export const createResourceAndMultipartUpload = async (
+  file: UppyFile,
+  onCreateFile: (newResource: Resource) => void
+) => {
   const createResourceResponse = await createResource(ResourceCreationType.FILE, file.name);
   const contentId = createResourceResponse.data.contents[0].identifier;
-  onCreateFile(createResourceResponse.data.identifier);
+  const newFile: Content = {
+    identifier: contentId,
+    features: {
+      dlr_content_title: file.name,
+      dlr_content_size: '' + file.size,
+      dlr_content_mime_type: '' + file.type,
+      dlr_content_master: 'true',
+    },
+  };
+  const newResource: Resource = {
+    identifier: createResourceResponse.data.identifier,
+    features: {
+      dlr_title: file.name,
+    },
+    contents: [newFile],
+  };
+  onCreateFile(newResource);
```

Comment:
må innrømme jeg ikke henger helt med på hva som skjer og hvorfor her. `createResource` er greit nok, `newFile` er sikkert for å gjøre om på fila til uppy for at det skal passe med backend.  `newResource` er litt mystisk, for jeg ville trodd at responsen fra `createResource` var den nye ressursen... men jeg vet ikke hvordan backend osv ser ut, så kan godt være det gir mening

Reply:
her har jeg knotet, ja - fikser det i en egen pr